projects/aca-content/src/lib/components/search/search-results/search-results.component.scss

```diff
@@ -108,6 +108,7 @@ aca-search-results {
 
     &--visibility_button {
       margin-right: 8px;
+      cursor: default;
```

Comment:
the mat-button should already have a cursor on it, so this might not be needed.. you can also remove the entire class, if setting for `adf-search-results--preview-toolbar` the `column-gap: 8px`, as the button should not be setting own margin here

Reply:
The cursor is set to `pointer` by default, but in this case I want to set it to `default`, because this button indicates that preview mode is active but has no action when preview is open.
![Screenshot 2024-08-19 at 08 46 21](https://github.com/user-attachments/assets/f99066d3-59eb-4523-b00c-7b08ce895b1b)